api/src/main/java/org/pmiops/workbench/cloudtasks/TaskQueueService.java

```diff
@@ -126,7 +127,7 @@ public List<String> groupAndPushSynchronizeAccessTasks(List<Long> userIds) {
           createAndPushTask(
               SYNCHRONIZE_ACCESS_QUEUE_NAME,
               SYNCHRONIZE_ACCESS_PATH,
-              new AuditProjectAccessRequest().userIds(group)));
```

Comment:
This **shouldn't** work, but it did because the JSON serializations of these are identical.